staging/src/k8s.io/component-base/config/v1alpha1/defaults.go

```diff
@@ -126,3 +126,18 @@ func RecommendedLoggingConfiguration(obj *LoggingConfiguration) {
 		obj.FlushFrequency = 5 * time.Second
 	}
 }
+
+// RecommendedTracingConfiguration defaults tracing configuration.
+// This will set the recommended default
+// values, but they may be subject to change between API versions. This function
+// is intentionally not registered in the scheme as a "normal" `SetDefaults_Foo`
+// function to allow consumers of this type to set whatever defaults for their
+// embedded configs. Forcing consumers to use these defaults would be problematic
+// as defaulting in the scheme is done as part of the conversion, and there would
+// be no easy way to opt-out. Instead, if you want to use this defaulting method
+// run it in your wrapper struct of this type in its `SetDefaults_` method.
+func RecommendedTracingConfiguration(obj *TracingConfiguration) {
+	if obj.Endpoint == nil {
+		obj.Endpoint = utilpointer.String("localhost:4317")
```

Comment:
Can we just rely on the fact that otel's default is already :4317, and not set our own value here?

Reply:
yes, removed this and the call to it
from godoc
`If unset, it will instead try to use connect to DefaultCollectorHost:DefaultCollectorPort` localhost:4317